.github/issue_template.md

```diff
@@ -1 +1,10 @@
+<!-- New Contributor? Welcome!
```

Comment:
Creating an issue will also add notes to your GitHub history, so it's important are aware of these guidelines for issues as well as pull requests.